src/main/java/net/consensys/orion/utils/OrionInfo.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package net.consensys.orion.utils;
+
+public class OrionInfo {
+
+  private static final String CLIENT = "orion";
+  private static final String VERSION = OrionInfo.class.getPackage().getImplementationVersion();
+  private static final String OS = PlatformDetector.getOS();
+  private static final String VM = PlatformDetector.getVM();
+
+  private OrionInfo() {}
+
+  public static String version() {
+    return String.format("%s/%s/%s/%s", CLIENT, VERSION != null ? "v" + VERSION : "development", OS, VM);
```

Comment:
The version string can be the following:
- `orion/v1.6.0/osx-x86_64/oracle_openjdk-java-11` when using a built version of Orion
- `orion/development/osx-x86_64/oracle_openjdk-java-11` when running it in the IDE for example